libcxx/utils/ci/run-buildbot

```diff
@@ -226,13 +231,8 @@ function test-armv7m-picolibc() {
 
     step "Installing compiler-rt"
     ${NINJA} -vC "${BUILD_DIR}/compiler-rt" install
-
-    # Prior to clang 19, armv7m-none-eabi normalised to armv7m-none-unknown-eabi.
-    # clang 19 changed this to armv7m-unknown-none-eabi. So for as long as 18.x
-    # is supported, we have to ask clang what the triple will be.
-    NORMALISED_TARGET_TRIPLE=$(${CC-cc} --target=armv7m-none-eabi -print-target-triple)
-    # Without this step linking fails later in the build.
-    mv "${BUILD_DIR}/install/lib/${NORMALISED_TARGET_TRIPLE}"/* "${BUILD_DIR}/install/lib"
+    # Move compiler-rt libs into the same directory as all the picolib objects.
+    mv "${BUILD_DIR}/install/lib/armv7m-unknown-none-eabi"/* "${BUILD_DIR}/install/lib"
```

Comment:
Looking at the script this looks like a portable solution. We no longer depend on the exact value of INSTALL_DIR.
```suggestion
    mv "${INSTALL_DIR}/lib/armv7m-unknown-none-eabi"/* "${INSTALL_DIR}/lib"
```

Reply:
Done.